extensions/src/ACRE2Core/Engine.cpp

```diff
@@ -74,15 +74,15 @@ ACRE_RESULT CEngine::initialize(IClient *client, IServer *externalServer, std::s
     return ACRE_OK;
 }
 
-ACRE_RESULT CEngine::initialize(IClient *client, IServer *externalServer, std::string fromPipeName, std::string toPipeName, std::string loggingPath) {
+ACRE_RESULT CEngine::initialize(IClient *const client, IServer *const externalServer, const std::string fromPipeName, const std::string toPipeName, const std::string loggingPath) {
 
-    g_Log = (Log *)new Log(const_cast<char *>(loggingPath.c_str()));
+    g_Log = (Log *) new Log(const_cast<char *>(loggingPath.c_str()));
```

Comment:
Why is the reason of casting `Log*` to `Log*` here?

Reply:
And why is it casting a constant string to a non constant char*, That might be undefined behaviour.